backend/api/search.py

```diff
@@ -84,34 +84,51 @@ def search_avanced():
 
     try:
         search_string = query['searchString']
```

Comment:
do we really want "searchString" in the query? I feel like we should definitely be consistent with our other json key naming